python/lsst/meas/extensions/trailedSources/NaivePlugin.py

```diff
@@ -191,6 +192,24 @@ def measure(self, measRecord, exposure):
         x1 = xc + dydtheta
         y1 = yc + dxdtheta
 
+        if not (exposure.getBBox().beginX <= x0 <= exposure.getBBox().endX
```

Comment:
Add one-line comment, e.g., `# Check whether trail extends into the edge of the exposure`

Reply:
Added